core/src/main/scala/org/apache/spark/sql/rapids/tool/util/ToolsPlanGraph.scala

```diff
@@ -290,7 +356,8 @@ object ToolsPlanGraph {
           planInfo.nodeName,
           planInfo.simpleString,
           mutable.ArrayBuffer[SparkPlanGraphNode](),
-          metrics)
+          metrics,
+          isPhotonNode = isPhotonNode)
```

Comment:
We don't need to have `isPhotonNode` as argument because it is set using an object's method. Then have different classes for photonNodes.

Reply:
Removed addition of `isPhotonNode` as we have different classes for PhotonNodes